src/updater-builder.ts

```diff
@@ -26,9 +26,8 @@ export class UpdaterBuilder {
   ) {}
 
   async run(containerName: string): Promise<Container> {
-    const cmd = `(echo > /etc/ca-certificates.conf) &&\
-     rm -Rf /usr/share/ca-certificates/ &&\
```

Comment:
Do we need to remove this directory still?

Reply:
I removed them because the dependabot user doesn't have write access to delete them. I could go back to the core image and make them writable, but the `/usr/share/ca-certificates` just contains some system-wide certs so I'm not sure what the benefit is of deleting them is, so I just left them.